docs/search/search_in_trash_reference.md

```diff
@@ -5,13 +5,14 @@ page_type: reference
 
 # Search in trash reference
 
-When you [search for content items that are held in trash](search_api.md#searching-in-trash), you can apply only a limited set of Search Criteria and Sort Clauses.
-They can be used by `Ibexa\Contracts\Core\Repository\TrashService::findTrashItems` only.
+When you [search for content items that are held in trash](search_api.md#searching-in-trash), you can apply only a limited subset of Search Criteria and Sort Clauses
+which can be used by [`Ibexa\Contracts\Core\Repository\TrashService::findTrashItems`](../api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-TrashService.html#method_findTrashItems).
+Some sort clauses are exclusive to trash search.
 
 ## Search Criteria
 
 - [ContentTypeId](contenttypeid_criterion.md)
```

Comment:
Shouldn't Content Name be listed here as well?

Reply:
Added in 01a03dddf6f2551a3168d8efe11606ae8d3e8b8b